src/main/java/hyperledger/besu/java/rest/client/service/impl/EventPublishServiceImpl.java

```diff
@@ -72,4 +78,62 @@ public void onFailure(Throwable ex) {
 
     return status;
   }
+
+  @Override
+  public boolean publishEventLogs(Object eventLog) {
+    boolean status = true;
+    ProducerRecord<String, String> producerRecord;
+    try {
+
+      Class<?> clazz = eventLog.getClass();
+
+      if (Log.class.equals(clazz)) {
+        assert eventLog instanceof Log;
```

Comment:
Why do we need assert? Why not just check `eventLog instanceof Log` in the `if` condition ?

Reply:
corrected